source/nodejs/tests/ui-tests/tslint.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
`tslint` has been deprecated in favor of `eslint` with the `typescript-eslint` plugin... we already have a config for that here:

https://github.com/microsoft/AdaptiveCards/blob/main/source/nodejs/.eslintrc.js